tests/services/inventory/test_data/additional_cai_resources.dump

```diff
@@ -60,6 +60,20 @@
9},{"names":["gcr.io/google_containers/heapster-amd64:v1.5.2"],"sizeBytes":75318342},{"names":["gcr.io/google_containers/heapster-amd64:v1.4.3"],"sizeBytes":73412845},{"names":["gcr.io/google_containers/heapster-amd64:v1.3.0"],"sizeBytes":68105973},{"names":["gcr.io/google-containers/prometheus-to-sd:v0.2.3"],"sizeBytes":55317339},{"names":["gcr.io/google-containers/prometheus-to-sd:v0.2.2"],"sizeBytes":55207051},{"names":["k8s.gcr.io/k8s-dns-kube-dns-amd64:1.14.13"],"sizeBytes":51152998},{"names":["gcr.io/google_containers/cluster-proportional-autoscaler-amd64:1.1.2-r2"],"sizeBytes":49644153},{"names":["k8s.gcr.io/k8s-dns-kube-dns-amd64:1.14.10"],"sizeBytes":49545125},{"names":["gcr.io/google_containers/k8s-dns-kube-dns-amd64:1.14.4"],"sizeBytes":49383112},{"names":["gcr.io/google_containers/k8s-dns-kube-dns-amd64:1.14.5"],"sizeBytes":49383083},{"names":["gcr.io/google_containers/cluster-proportional-autoscaler-amd64:1.1.1-r3"],"sizeBytes":47327049},{"names":["gcr.io/google_containers/kubedns-amd64:1.9"],"sizeBytes":46998769},{"names":["k8s.gcr.io/k8s-dns-sidecar-amd64:1.14.13"],"sizeBytes":42847643},{"names":["gcr.io/google_containers/metrics-server-amd64:v0.2.1"],"sizeBytes":42541759},{"names":["k8s.gcr.io/prometheus-to-sd:v0.4.2"],"sizeBytes":41844867},{"names":["gcr.io/google_containers/k8s-dns-sidecar-amd64:1.14.4"],"sizeBytes":41814878},{"names":["gcr.io/google_containers/k8s-dns-sidecar-amd64:1.14.5"],"sizeBytes":41814849},{"names":["k8s.gcr.io/k8s-dns-sidecar-amd64:1.14.10"],"sizeBytes":41630977},{"names":["gcr.io/google_containers/k8s-dns-dnsmasq-nanny-amd64:1.14.5"],"sizeBytes":41419289},{"names":["gcr.io/google_containers/k8s-dns-dnsmasq-nanny-amd64:1.14.4"],"sizeBytes":41410758},{"names":["k8s.gcr.io/k8s-dns-dnsmasq-nanny-amd64:1.14.13"],"sizeBytes":41368100},{"names":["k8s.gcr.io/k8s-dns-dnsmasq-nanny-amd64:1.14.10"],"sizeBytes":40367821},{"names":["gcr.io/google_containers/addon-resizer:1.7"],"sizeBytes":38983736},{"names":["gcr.io/google_containers/addon-resizer:1.8.1","k8s.gcr.io/addon-resizer:1.8.1"],"sizeBytes":32968591},{"names":["gcr.io/google_containers/dnsmasq-metrics-amd64:1.0"],"sizeBytes":13998769},{"names":["gcr.io/google_containers/exechealthz-amd64:1.2"],"sizeBytes":8374840},{"names":["gcr.io/google_containers/defaultbackend:1.0"],"sizeBytes":7510068},{"names":["k8s.gcr.io/defaultbackend-amd64:1.5"],"sizeBytes":5132544},{"names":["gcr.io/google_containers/kube-dnsmasq-amd64:1.4"],"sizeBytes":5126001},{"names":["gcr.io/google_containers/defaultbackend:1.3"],"sizeBytes":3618880},{"names":["gcr.io/google_containers/pause-amd64:3.0"],"sizeBytes":746888}],"nodeInfo":{"architecture":"amd64","bootID":"89432ad0-2e7f-42d7-b7e8-b7089ef3c7f0","containerRuntimeVersion":"docker://1.11.2","kernelVersion":"4.4.21+","kubeProxyVersion":"v1.4.5","kubeletVersion":"v1.4.5","machineID":"22c572c5043e49e5baa8e5b458275987","operatingSystem":"linux","osImage":"Google Container-VM Image","systemUUID":"F9D60A54-DA66-E5EA-F675-D162D7C0C7A8"}}}}}
```

Comment:
Please remove non-test identifiers like the cluster name and replace with a generic cluster name (and below). Anything that came back from the API that isn't generic needs to be replaced.

Reply:
Done.